plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoClientConfig.java

```diff
@@ -35,6 +35,7 @@ public void testDefaults()
                 .setCaseInsensitiveNameMatching(false)
                 .setSeeds("")
                 .setCredentials("")
+                .setMetadataPrivilegesRequired(true)
```

Comment:
You should also put that add that property to `testExplicitPropertyMappings` method

Reply:
I removed property so this does not exists anymore